src/services/institution-service.js

```diff
@@ -12,7 +12,7 @@ async function fetchInstitutions(projectId) {
 
   let res = []
   for (var i in rows) {
-    res.push(rows[i].name)
+    res.push({ name: rows[i].name, institutionId: rows[i].institutionId })
   }
   return res
```

Comment:
I don't think that the loop is necessary. You can remove the `res` value and just return `rows`

Also can you update `rows` from `let` to `const` since the value doesn't change.